CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## [1.0.120221116]
```

Comment:
💡 minor question, why its version: ` [1.0.120221116]` ? and not ` [1.0.120221201]`